plugins/dicom_viewer/web_client/views/DicomView.js

```diff
@@ -0,0 +1,349 @@
+import ViewTemplate from '../templates/view.pug';
+import TagsTemplate from '../templates/tags.pug';
+import '../stylesheets/dicom_viewer.styl';
+
+import { restRequest, apiRoot } from 'girder/rest';
+import { wrap } from 'girder/utilities/PluginUtils';
+import ItemView from 'girder/views/body/ItemView';
+import View from 'girder/views/View';
+
+import daikon from 'daikon';
+import vtkImageSlice from 'vtk.js/Sources/Rendering/Core/ImageSlice';
+import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkImageMapper from 'vtk.js/Sources/Rendering/Core/ImageMapper';
+import vtkInteractorStyleImage from 'vtk.js/Sources/Interaction/Style/InteractorStyleImage';
+import vtkOpenGLRenderWindow from 'vtk.js/Sources/Rendering/OpenGL/RenderWindow';
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+import vtkRenderWindow from 'vtk.js/Sources/Rendering/Core/RenderWindow';
+import vtkRenderWindowInteractor from 'vtk.js/Sources/Rendering/Core/RenderWindowInteractor';
+
+wrap(ItemView, 'render', function (render) {
+    this.once('g:rendered', function () {
+        $('.g-item-header').after('<div id="g-dicom-view"></div>');
+        /* eslint-disable no-new */
+        new DicomView({
+            el: $('#g-dicom-view'),
```

Comment:
`el: this.$('#g-dicom-view')` (will work if you change the containing function to an arrow function)